src/js/core.js

```diff
@@ -1088,6 +1113,81 @@
                 target.innerHTML = html;
                 this.events.updateInput(target, { target: target, currentTarget: target });
             }
+        },
+
+        addElement: function (element) {
+            if (element.length) {
+                // it is already an array..
+                return this.addElements(element);
+            }
+
+            this.addElements([element]);
+        },
+
+        addElements: function (elements) {
+            var filtered = [];
+
+            // Filter the input, we want to include every element only once!
+            elements.forEach(function (element) {
+                if (element.getAttribute('data-medium-editor-element')) {
+                    return;
+                }
+
+                filtered.push(element);
+            });
+
+            // Do we have elements to add now?
+            if (filtered.length === 0) {
+                return false;
+            }
+
+            // Add new elements to our internal elements array
+            this.elements = this.elements.concat(filtered);
+
+            // Initialize all new elements (we check that in those functions don't worry)
+            initElements.call(this);
+            convertTextareas.call(this);
+
+            filtered.forEach(function (element) {
+                reAttachHandlers.apply(this, [element]);
+            }.bind(this));
+        },
+
+        removeElement: function (elementToRemove) {
+            if (elementToRemove.length) {
+                // it is already an array..
+                return this.removeElements(elementToRemove);
+            }
+
+            var filtered = [];
+
+            this.elements.forEach(function (element) {
+                if (element.getAttribute('medium-editor-index') !== elementToRemove.getAttribute('medium-editor-index')) {
+                    filtered.push(element);
+                } else {
+                    this.events.cleanupElement(element);
+                }
+            }.bind(this));
+
+            this.elements = filtered;
+        },
+
+        removeElements: function (elements) {
+            elements.forEach(function (element) {
+                this.removeElement(element);
+            }.bind(this));
+        },
+
+        guid: function () {
```

Comment:
How would you feel about moving this into `util.js`?  Just a way to reduce the number of methods exposed in the core API since it's not tied to the editor instance.